docs/concepts/pdp/overview.mdx

```diff
@@ -0,0 +1,248 @@
+
+For more information, see the [sidecar deployment guide](/how-to/deploy/overview#sidecar).
+
+```mermaid
+flowchart LR
+    subgraph Your Network
+        pdp[PDP Sidecar]
+        app[Your service] -- query --> pdp
+    end
+
+    cloud[Permit.io Cloud]
+    pdp --> cloud
+
+    style cloud fill:#ffb381,color:#000;
+    style app fill:#974ef2,color:#fff;
+    style pdp fill:#f76808,color:#fff;
+```
+
+### Centralized PDP
+
+You can deploy a centralized PDP to handle authorization queries from multiple services.
+
+This way is simple to set up and manage, but as the number of services grows, the PDP can become a bottleneck.
+
+```mermaid
+flowchart LR
+    subgraph Your Network
+        pdp[PDP]
+        app1[Service #1] -- query --> pdp
+        app2[Service #2] -- query --> pdp
+    end
+
+    cloud[Permit.io Cloud]
+    pdp --> cloud
+
+    style cloud fill:#ffb381,color:#000;
+    style app1 fill:#974ef2,color:#fff;
+    style app2 fill:#974ef2,color:#fff;
+    style pdp fill:#f76808,color:#fff;
+```
+
+### Cluster of PDPs
+
+For high availability and scalability, you can deploy a cluster of PDPs behind a load balancer.
+
+You can use our PDP Helm chart to create a Kubernetes deployment with multiple PDPs.
+For more information, see the [Deploying using Helm guide](/how-to/deploy/cloud-hosts/helm).
+
+```mermaid
+flowchart LR
+    subgraph Your Network
+        lb[Load Balancer]
+        pdp1[PDP]
+        pdp2[PDP]
+        app1[Service #1] -- query --> lb
+        app2[Service #2] -- query --> lb
+        lb --> pdp1
+        lb --> pdp2
+    end
+
+    cloud[Permit.io Cloud]
+    pdp1 --> cloud
+    pdp2 --> cloud
+
+    style cloud fill:#ffb381,color:#000;
+    style app1 fill:#974ef2,color:#fff;
+    style app2 fill:#974ef2,color:#fff;
+    style pdp1 fill:#f76808,color:#fff;
+    style pdp2 fill:#f76808,color:#fff;
+    style lb fill:#4e7ef2,color:#fff;
+```
+
+### Sharded PDPs
+
+For truly massive data sets needed in the PDP, you can apply sharding to split the data between multiple PDPs within the same cluster.
+
+This is done by using an [Envoy Server](https://www.envoyproxy.io/docs/envoy/latest/) as proxy to route requests to the relevant PDP based on the tenant.
+For more information, see the [Sharded PDPs guide](/how-to/manage-data/pdp-sharding).
+
+```mermaid
+graph LR
+    subgraph Your Network
+        envoy[Envoy]
+        pdp1[PDP - Shard #1]
+        pdp2[PDP - Shard #2]
+        app1[Service #1] -- query --> envoy
+        app2[Service #2] -- query --> envoy
+        envoy -->|Based on Tenant| pdp1
+        envoy -->|Based on Tenant| pdp2
+    end
+
+    cloud[Permit.io Cloud]
+    pdp1 --> cloud
+    pdp2 --> cloud
+
+    style cloud fill:#ffb381,color:#000;
+    style app1 fill:#974ef2,color:#fff;
+    style app2 fill:#974ef2,color:#fff;
+    style pdp1 fill:#f76808,color:#fff;
+    style pdp2 fill:#f76808,color:#fff;
+    style envoy fill:#4e7ef2,color:#fff;
+```
+
+## Powered by OPA+OPAL
```

Comment:
same here, I think we would want to change the powered by OPA+OPAL as we will support more engines in the future.
@orweis @obsd WDYT ?